apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -304,6 +306,7 @@ export interface ResearchOutputController {
 }
 export type ResearchOutputCreateData = ResearchOutputPostRequest & {
   createdBy: string;
+  addedDate?: string;
```

Comment:
the `ResearchOutputCreateData` is the type that the controller accepts as input, and here:
https://github.com/yldio/asap-hub/blob/e5eac73a96c54e8d705c26cd8cf699fa39d2e9d4/apps/crn-server/src/controllers/research-outputs.ts#L92
you are telling the controller to fill the date for  data-provider when it's not present. The thing is though - as you removed this property from the payload it will never be available anyway. I would say that you should remove this property from the `ResearchOutputCreateData` and only keep it in `ResearchOutputCreateDataObject`